505_output_coverage_jsons.sql.tpl

```diff
@@ -0,0 +1,66 @@
+    begin;
+    -- move this to its own block if it works
+ select (select release from audit_event order by release limit 1) as latest_release
+ \gset
+ \set output_file 'resources/coverage/{{ .UnreleasedVersion }}.json'
```

Comment:
import contents of plain text file, using psql instead of using Go templating.
Write contents to a file somewhere.